enos/ci/aws-nuke.yml

```diff
@@ -0,0 +1,409 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: BUSL-1.1
```

Comment:
I believe all files in this repo should be `MPL-2.0` since it is a community repo.